modules/standard/Map.chpl

```diff
@@ -317,10 +374,13 @@ module Map {
     }
 
     /* Get a reference to the element at position `k`. This method is not
-       available for non-nilable types.
+       available for maps initialized with `parSafe=true`.
      */
-    proc getReference(k: keyType) ref
-    where !isNonNilableClass(valType) {
```

Comment:
Did something change to make this work on non-nilable classes when `parSafe=false`?

Reply:
I think that since we implement map using `chpl__hashtable`, non-nilable classes should be supported now. But I will add a test to make sure.